sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/windowExpressions.scala

```diff
@@ -426,8 +427,16 @@ abstract class OffsetWindowFunction
       * default - a string expression which is to use when the offset is larger than the window.
           The default value is null.
   """,
+  examples = """
+    Examples:
+      > SELECT a, b, _FUNC_(b) OVER (PARTITION BY a ORDER BY b) FROM VALUES ('A1', 2), ('A2', 3), ('A1', 1) tab(a, b);
+       A1	1	2
+       A1	2	NULL
+       A2	3	NULL
```

Comment:
Can we just we whitepsaces instead? I think that's what other examples do IIRC.

Reply:
Do you mean using spaces instead of tabs?
The `check outputs of expression examples` in `ExpressionInfoSuite` does a string comparison on requires a tab to be the column separator. 
All the other examples I checked also use tabs.